doc/INSTALL-Win64.txt

```diff
@@ -0,0 +1,178 @@
+-------------------------------------------------------------------------------
+KVIrc4 win64 compiling howto                                          by Cizzle
+ based on wodim's guide at https://thacid.wordpress.com/2010/10/18/compiling-kvirc-64-bits-from-svn-using-visual-studio-2008/
+ and DarthGandalf's appveyor buildinstructions
+-------------------------------------------------------------------------------
+
+This paper describes how to successfully compile a working copy of KVIrc 4.3.x
+under Microsoft Windows operating systems. For own compilations we assume a
+root-directory C:\kvirccomp\ being used.
+
+-------------------------------------------------------------------------------
+Prerequisites
```

Comment:
Many of them are easier to be installed using chocolatey, though manually downloading them and running setup.exe (or whatever) works too, of course.